assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -275,6 +325,18 @@ class PageEdit extends Component {
 					</PanelColorSettings>
 					<PanelBody title={ __( 'Background Media', 'amp' ) }>
 						<>
+							{
+								isExcessiveVideoSize &&
+								<Notice status="warning" isDismissible={ false } >
+									{
+										sprintf(
+											/* translators: %d: the number of bytes in a megabyte */
+											__( 'The video size is more than %d bytes per second.', 'amp' ),
```

Comment:
As Weston already mentioned, this should still use MB in the string, so with something like `MEGABYTE_IN_BYTES / 1024`.

For featured image we say `Selecting a featured image is recommended for an optimal user experience.` (see `validateFeaturedImage()`). As a user I'd expect a similar user-friendly message for large videos.

Reply:
Ah, thanks. How does https://github.com/ampproject/amp-wp/pull/2509/commits/c966459de356d8d4280eedb550ce65bfae71fcbe look?

<img width="388" alt="new-notice-text" src="https://user-images.githubusercontent.com/4063887/59139598-aed80c80-8959-11e9-82dc-4fb4a36c31ab.png">

It doesn't use `1024` in the calculation, though. Is that alright?

Reply:
Much better this way 👍

Reply:
Thanks!